3ds2playground/src/main/kotlin/com/stripe/android/stripe3ds2playground/MainActivity.kt

```diff
@@ -471,7 +472,9 @@ class MainActivity : AppCompatActivity() {
                     acsTransId = acsTransId,
                     threeDsServerTransId = threeDsServerTransId,
                     sdkTransId = SDK_TRANS_ID,
-                    messageVersion = "2.1.0"
+                    messageVersion = "2.2.0",
+                    threeDSRequestorAppURL = "threeDSRequestorAppURL"
+
```

Comment:
is it intentional that this app url would be different from the one above?

also there's an extra blank line here, I'm kinda surprised this passed our lint check. Or is the lint check not running on this module? Probably not a big deal but just flagging

Reply:
Hmm yeah good question, this is just the playground so I think we can leave it